erpnext/erpnext_integrations/doctype/plaid_settings/plaid_settings.py

```diff
@@ -264,7 +264,7 @@ def new_bank_transaction(transaction):
 					"reference_number": (
 						transaction["check_number"]
 						or transaction["payment_meta"]["reference_number"]
-						or transaction["name"]
+						or transaction["name"][:20]
```

Comment:
```suggestion
						or transaction["name"][:140]
```

Reply:
convert so small text makes no sense; that would just lead to double information storage. 20 was chosen because of standard international banking check length. addition of [...] would be good though, want me to add that ?
btw. removing [name] is also an option. in v13 this field was also empty.